org.eclipse.jdt.core.javac/src/org/eclipse/jdt/core/dom/JavacProblemConverter.java

```diff
@@ -0,0 +1,113 @@
+/*******************************************************************************
+* Copyright (c) 2024 Microsoft Corporation and others.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-2.0/
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     Microsoft Corporation - initial API and implementation
+*******************************************************************************/
+
+package org.eclipse.jdt.core.dom;
```

Comment:
Same

Reply:
Yes, the problem converter can be moved to internal.